test.py

```diff
@@ -61,7 +68,45 @@
 counter = 0
 for comp, decomp in zip(compressed, decompressed):
     for en, de in zip(comp, decomp):
-        en = en.astype('float16')
+        de = np.squeeze(de)
         np.savez(save_path/f'compressed_{counter}', data=en)
         np.savez(save_path/f'decompressed_{counter}', data=de)
         counter += 1
+
+
+#=================== Load result and compute metrices ===================
```

Comment:
Hi @pphuangyi  

We need to add a test on the difference between our decompressed data vs other vendor created decompressed data. 
Can you do:
1) package the compressed samples and decompressed samples into the `sample.tgz` as well?
2) add a test to print out the difference between our saved compressed code  (in `data`) vs vendors' compressed code (in `results`)?